kubernetes/ingress/recycledelec.com.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: certmanager.k8s.io/v1alpha1
```

Comment:
I guess we actually don't need this one in Kubernetes since it's hosted externally (on Tumblr by the looks of it; the 66.6.44.4 IP in the DNS is Tumblr).

Reply:
Ahh, didn't realize those were the only entries in the DNS list.